### ISSUE TITLE
5.40 crashes with System.FormatException, Please manually upgrade to 5.41 ASAP!

### ISSUE THREAD

**Issue by hjjiang** [kangyu-california/PersistentWindows#213]
Application: PersistentWindows.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.FormatException
   at System.Number.StringToNumber(System.String, System.Globalization.NumberStyles, NumberBuffer ByRef, System.Globalization.NumberFormatInfo, Boolean)
   at System.Number.ParseInt32(System.String, System.Globalization.NumberStyles, System.Globalization.NumberFormatInfo)
   at Ninjacrab.PersistentWindows.SystrayShell.SystrayForm.CheckUpgrade()
   at Ninjacrab.PersistentWindows.SystrayShell.SystrayForm.UpdateMenuEnable(Boolean, Boolean)
   at Ninjacrab.PersistentWindows.Common.PersistentWindowProcessor.Start(Boolean)
   at Ninjacrab.PersistentWindows.SystrayShell.Program.Main(System.String[])

**Comment by kangyu-california** [comment]
github suddenly changed the format of release page
Please manually upgrade to 5.41

**Comment by jdreinhardt** [comment]
I was also experiencing this error on startup. Everything is working again after a manual upgrade to 5.41.

**Comment by hjjiang** [comment]
THANKYOU!

**Comment by kangyu-california** [comment]
reopen to avoid duplicated ticket.

**Comment by yrogerg77** [comment]
Thanks! I was hoping there was a fix available... Using Windows with two different sized monitors is hell without this program!

**Comment by CrushU** [comment]
Confirming that the new update did fix the issue. Thank you.

**Comment by rongyaota** [comment]
Sorry to ask. How do we "manually upgrade to 5.41 ASAP!"?

**Comment by kangyu-california** [comment]
> Sorry to ask. How do we "manually upgrade to 5.41 ASAP!"?

Just unzip https://github.com/kangyu-california/PersistentWindows/releases/download/5.41/PersistentWindows5.41.zip over  your old installation

**Comment by rongyaota** [comment]
> > Sorry to ask. How do we "manually upgrade to 5.41 ASAP!"?
> 
> Just unzip https://github.com/kangyu-california/PersistentWindows/releases/download/5.41/PersistentWindows5.41.zip over your old installation

Wonderful! Fixed the problem immediately of app crashing.
Can I suggest you add a link on the main page for installation steps? I was digging around but couldn't find one. I saw this but couldn't remember how I had originally installed so was reluctant.
And it's a wonderful tool. Just what I needed for multiple monitors - especially on a laptop/external monitor.

I'm interested in other apps you have or develop.
